torch/lib/c10d/ProcessGroupGloo.hpp

```diff
@@ -69,12 +69,16 @@ class ProcessGroupGloo : public ProcessGroup {
   // operations using the new AsyncWork base class. Over time we will port
   // all operations and perform needed cleanup.
   //
+  // FIXME: This probably should be called WorkGloo since the work is executed in sync mode
+  // by a background thread.
   class AsyncWork : public ProcessGroup::Work {
    public:
     AsyncWork(
+        std::vector<std::vector<at::Tensor>> outputTensors,
```

Comment:
Will this break backward compatibility?

Reply:
no, work constructors aren't part of public facing API.